helm/istio_cni/templates/daemonset.yaml

```diff
@@ -74,21 +74,20 @@ spec:
               value: "/host/etc/cni/multus/net.d"
             # Name of the CNI config file to create.
             - name: CNI_CONF_NAME
-              value: "{{ .Release.Namespace }}-istio-cni.conf"
+              value: "istio-cni.conf"
             # Name of the kubeconfig file used by CNI agent
             - name: KUBECFG_FILE_NAME
-              value: "{{ .Release.Namespace }}-istio-cni.kubeconfig"
-            # Prefix for the istio-cni binary and install-iptables.sh script
+              value: "istio-cni.kubeconfig"
             - name: CNI_BINARIES_PREFIX
-              value: "{{ .Release.Namespace }}-"
+              value: ""
```

Comment:
Can we just remove this?

Reply:
We can, once my Istio CNI PR that fixes this gets merged. Until then, I wanted to make sure these changes work properly.